.goreleaser.yml

```diff
@@ -16,10 +16,6 @@ builds:
   # Get path to OperatorVersion using: $(go list -m)
   - -X github.com/goharbor/harbor-operator.OperatorVersion={{ if .Tag }}{{ .Tag }}{{ else }}dev{{ end }}
 
-dockers:
-- image_templates:
-  - holyhope/harbor-operator:{{ .Tag }}
```

Comment:
I suggest that we keep using `dockers`, but to build the `goharbor/harbor-operator` image.
<https://goreleaser.com/customization/docker/>
My goal is to have all release configuration in a single file.
WDYT?

Reply:
ok